tasks/winbuildscripts/Generate-Chocolatey-Package.ps1

```diff
@@ -64,7 +64,7 @@ if (!(Test-Path $outputDirectory)) {
 
 if ($installMethod -eq "online") {
     # Set the $url in the install script
-    (Get-Content $installScript).replace('$url', $url) | Set-Content $installScript
+    (Get-Content $installScript).replace('__url_from_ci__', '"' +  $url  + '"') | Set-Content $installScript
```

Comment:
I think it should be
```suggestion
    (Get-Content $installScript).replace('$__url_from_ci__', '"' +  $url  + '"') | Set-Content $installScript
```
but that will also replace it in the comment you added 😅